src/jit/lsraarmarch.cpp

```diff
@@ -718,6 +732,57 @@ void Lowering::TreeNodeInfoInitPutArgStk(GenTreePutArgStk* argNode, fgArgTabEntr
     }
 }
 
+#ifdef _TARGET_ARM_
+//------------------------------------------------------------------------
+// TreeNodeInfoInitPutArgStruct: Set the NodeInfo for a GT_PUTARG_STRUCT node
+//
+// Arguments:
+//    argNode - a GT_PUTARG_STRUCT node
+//
```

Comment:
This comment needs to be updated (it uses "struct" instead of "split" several places.)